octobot_trading/api/symbol_data.py

```diff
@@ -44,3 +44,7 @@ def create_new_candles_manager(candles=None) -> CandlesManager:
     if candles is not None:
         manager.replace_all_candles(candles)
     return manager
+
+
+def force_set_mark_price(exchange_manager, symbol, price):
```

Comment:
What will be the usage of this method ?

Reply:
Setting the price for testing ordre creation (trading mode tests)

Reply:
OK if it's only for testing purposes :+1: